components/authz-service/storage/v2/rule.go

```diff
@@ -52,24 +52,10 @@ func NewRule(id string, projectID string, name string,
 	}, nil
 }
 
-func validateRuleInputs(id string,
-	projectID string, name string, ruleType RuleType, conditions []Condition) error {
-
-	if emptyOrWhitespaceOnlyRE.MatchString(id) {
-		return errors.New(
-			"a rule id is required and must contain at least one non-whitespace character")
-	}
-	if emptyOrWhitespaceOnlyRE.MatchString(name) {
-		return errors.New(
-			"a rule name is required and must contain at least one non-whitespace character")
-	}
-	if emptyOrWhitespaceOnlyRE.MatchString(projectID) {
-		return errors.New("a rule id must have an associated project_id")
-	}
+func validateConditionsForType(ruleType RuleType, conditions []Condition) error {
 	if len(conditions) == 0 {
-		return errors.New("a rule must contain at least one condition")
+		return errors.New("at least one condition is required")
```

Comment:
should this move up to the server code, since that where the other required fields are being checked?